src/guide/composition-api-introduction.md

```diff
@@ -251,7 +251,7 @@ setup (props) {
     repositories.value = await fetchUserRepositories(props.user)
   }
 
-  onMounted(getUserRepositories) // on `mounted` call `getUserRepositories`
+  onMounted(getUserRepositories) // al "montarse" invoca `getUserRepositories`
```

Comment:
What do you think about this? This is to keep it consistent with the original docs.

```suggestion
  onMounted(getUserRepositories) // al `montarse` invoca `getUserRepositories`
```